popup.html

```diff
@@ -1,17 +1,18 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>WakaTime</title>
-
-    <link href="public/css/app.css" rel="stylesheet">
-</head>
-<body>
-
-<div id="wakatime"></div>
-
-<script src="public/js/app.js"></script>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WakaTime</title>
+
+    <link href="public/css/app.css" rel="stylesheet">
+    <script src="public/js/browser-polyfill.min.js"></script>

```

Comment:
Usually JavaScript is loaded after the DOM, so it doesn't try to modify the DOM before it's loaded. However, this polyfill shouldn't depend on the DOM so it's fine in the head.